### PR TITLE
commands in terminal most used added

### PR DESCRIPTION
**Pull request by abrilmunozc**

### CODE REVIEW

File: 01/index.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+pwd - para mostrar la ruta 
+open - para abrir finder
+mv - para mover archivos
+git init - para iniciar un repositorio en git
+code .. - abre visual studio
+command + s - para guardar cambios
+git add . - para agregar los archivos que han tenido cambios
+git commit -m "___" - para agregar un commit
+git branch -v muestra el nombre de la rama y el último commit
+git push -u origin "nombre de la rama" - para enviar o subir cambios a github
```
Comment (eamzea) [inline 01/index.html:22]: There are mixed examples of git and terminal and the exercise is just requesting terminal commands.

Comment (eamzea) [inline 01/index.html:22]: The instruction says to write a resume answering two requests:
https://github.com/zinns/HTML-Course/blob/5fa068ee9b4dfe15f48f02218c99b841f0ad6cfb/01/intro.md?plain=1#L13

**Review by soyedgarzea — CHANGES_REQUESTED** [review]

File: 01/index.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+¿Qué es la terminal? 
+Se utiliza una terminal para poder interactuar con un ordenador a través de comandos o líneas de texto y poder hacer diversas funciones.
```
Comment (eamzea) [inline 01/index.html:10]: This question is not requested in the instructions

File: 01/index.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+¿Cuáles son los comandos más utilizados?
+mkdir, touch, ls, pwd, open, mv, code .., command+s, entre otros.
```
Comment (eamzea) [inline 01/index.html:13]: - code .. is not a command
- command + s is not a command is a shortcut

File: 01/index.html [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+Ejemplos y descripción de comandos:
+mkdir - para crear una carpeta
+touch - para crear un archivo
+ls - para ver lo que hay dentro de una carpeta
+pwd - para mostrar la ruta 
+open - para abrir finder
+mv - para mover archivos
+code .. - abre visual studio
```
Comment (eamzea) [inline 01/index.html:22]: Command is wrong

File: 01/index.html [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+command + s - para guardar cambios
```
Comment (eamzea) [inline 01/index.html:23]: This is not a command is a shortcut

**Review by soyedgarzea — CHANGES_REQUESTED** [review]

File: 01/index.html (hunk 3 above)
Comment (eamzea) [reply 01/index.html:10]: I don't see any changes on this :|

**Review by soyedgarzea — CHANGES_REQUESTED** [review]
Branch has conflicts

**Review by soyedgarzea — CHANGES_REQUESTED** [review]
Branch has conflicts

**Review by soyedgarzea — APPROVED** [review]

**Pull request merged**